README.md

```diff
@@ -216,11 +216,11 @@ Setting the variable `dhcp_pxeboot_server`, will redirect PXE clients to the spe
 
 ### Custom Includes
 
-Setting the variable `dhcp_custom_inludes` to a jinja template will allow custom configurations to be used which will subsiquently be included into the `dhcpd.conf` file. 
+Setting the variable `dhcp_custom_inludes` to a jinja template will allow custom configurations to be used which will subsiquently be included into the `dhcpd.conf` file. If the template file name has the `.j2` extension it will be removed from the destination file name, else it will preserve the template file name in the destination.
 
 ```Yaml
 dhcp_custom_includes:
-  - custom-dhcp-config.conf
+  - custom-dhcp-config.conf.j2
```

Comment:
Because the file may not always be a template, this could be written as 

custom-dhcp-config.conf[.j2]

to show that the extension is optional.

Reply:
Also, there has been a recent push on master, it looks like you will need to merge master into your branch, before continuing....

Reply:
Ok. I will take a look at that.